Test-Json: document SchemaFile parameter (#6134)
MicrosoftDocs/PowerShell-Docs: reference/7.1/Microsoft.PowerShell.Utility/Test-Json.md

```yaml
Type: String
Parameter Sets: SchemaString
Aliases:

Required: False
Position: 2
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -SchemaFile

> Specifies a Schema file to validate the JSON input against. If passed `Test-Json` will validate that
> the Json input conforms to the spec specified by the **SchemaFile** parameter and return `$True`
> only if the input conforms to the provided Schema.

Reviewer: ```suggestion
Specifies a schema file used to validate the JSON input. When used, the `Test-Json` returns `$True`
only if the JSON input conforms to the Schema defined in the file specified by the **SchemaFile**
parameter.
```
